### ISSUE TITLE
the lfw acc

### ISSUE THREAD

**Issue by whcjb** [ronghuaiyang/arcface-pytorch#16]
hello, ronghuai, could tell the lfw acc you tested on your model, I use your training code, but can get only 98.n% acc, I have tuned almost all the super-parameters. but can not reach 99%

**Comment by whcjb** [comment]
could you tell the net backbone you used?

**Comment by ronghuaiyang** [comment]
the backbone i use is the same as sphereface, it's a 28 layer resnet. my dataset is cleaned cisia webface

**Comment by whcjb** [comment]
now, I see, thanks,

**Comment by luluvi** [comment]
> hello, ronghuai, could tell the lfw acc you tested on your model, I use your training code, but can get only 98.n% acc, I have tuned almost all the super-parameters. but can not reach 99%

How do you get the data set training, can you give me some ideas?
thanks!

**Comment by ronghuaiyang** [comment]
@luluvi if you use a public training set, you'd better clean the dataset.

**Comment by luluvi** [comment]
> @luluvi if you use a public training set, you'd better clean the dataset.

Is your cleaning done with face alignment, resize and then TXT format is the same as the test data you provided?
I am trying to do it, thanks for the code you shared.
I am honored to hear from you.

**Comment by ronghuaiyang** [comment]
@luluvi besides the image preprocess, you can also correct or just delete the wrong labeled images in the dataset.  In the public dataset there are a lot of wrong labeled images.

**Comment by luluvi** [comment]
> @luluvi besides the image preprocess, you can also correct or just delete the wrong labeled images in the dataset. In the public dataset there are a lot of wrong labeled images.

I am so happy to receive a reply on GitHub, I try my best to achieve it, thank you very much.

**Comment by ahuirecome** [comment]
Well, I cannot get a 99% accuracy either!
